src/components/docs-anchor/components/styled-anchor/index.tsx

```diff
@@ -0,0 +1,17 @@
+import classNames from 'classnames'
+import { AnchorHTMLAttributes } from 'react'
+import s from './styled-anchor.module.css'
+
+export default function StyledAnchor(
```

Comment:
I refactored the docs link to extract out just the styling of the anchor element so this could be shared. Another approach I considered was just `composing` the styles from the docs stylesheet in a custom component defined on the tutorials `MDX_COMPONENTS` list. I thought it might be nice to make this abstraction here, as just composing seems like it could introduce potential for drift. Super open to input here. 

This also made me realize that we could do more optimization for anchor links on tutorial content that is an internal versus external link and use `next/link` like in docs.

Reply:
I think with the goal being to have something reusable in many areas of the site, we should place this component in the main `src/components` file! ✨

We could name it something like `MDXInlineLink` which I think:
- aligns with how we want to name and create MDX components ([Convert global-ish docs content CSS to MDX custom components](Convert global-ish docs content CSS to MDX custom components))
- conveys it's intended use (in MDX content) & what component it's an abstraction of (`InlineLink`)

Reply:
Sounds good! i'll lift this up to there 👌